src/main/java/net/sf/rails/game/TrainCertificateType.java

```diff
@@ -206,6 +206,11 @@ public String getInitialPortfolio() {
         return initialPortfolio;
     }
 
+    public boolean isDual() {
+        if (potentialTrainTypes.size() == 2) return true;
```

Comment:
This could be written as thereby avoiding the if/else
```
return potentionalTrainTypes.size() == 2;
```

Reply:
done